src/main/scala/com/datastax/cdm/job/ConnectionFetcher.scala

```diff
@@ -72,6 +76,17 @@ class ConnectionFetcher(sparkContext: SparkContext, propertyHelper: PropertyHelp
         .set("spark.cassandra.auth.password", connectionDetails.password)
         .set("spark.cassandra.input.consistency.level", consistencyLevel)
         .set("spark.cassandra.connection.config.cloud.path", connectionDetails.scbPath))
+    } else if (connectionDetails.trustStorePath.nonEmpty && connectionDetails.isAstra) {
+      logger.info("Connecting to Astra "+side+" (with truststore) using host metadata at "+connectionDetails.host+":"+connectionDetails.port);
+
+      val scbBundle = generateSCB(connectionDetails.host, connectionDetails.port, 
```

Comment:
it's already a bundle.

```suggestion
      val scbFile = generateSCB(connectionDetails.host, connectionDetails.port, 
```